app/models/book.rb

```diff
@@ -2,4 +2,10 @@
 
 class Book < ApplicationRecord
   mount_uploader :picture, PictureUploader
+  belongs_to :user
+
+  def created_by?(user)
+    return false if user == nil
+    user_id == user.id
+  end
```

Comment:
これって還元していくとこうなって、

```suggestion
  def created_by?(target_user)
    user == target_user
  end
```

だったらviewの中で直接

```erb
<% if book.user == current_user %>
```

と書いたほうがわかりやすいってことになるかもです。